.snyk

```diff
@@ -0,0 +1,320 @@
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:03.042Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056416:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:05.978Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056417:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:10.245Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056418:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:13.473Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056419:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:16.350Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056420:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:21.665Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056421:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:25.685Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056424:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:28.835Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056425:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:33.200Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056426:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:36.453Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1056427:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:39.404Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-1061931:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:47.121Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-174736:
+    - '*':
+        reason: Runtime dependency by nexus and not using polymorphic type handling
+        expires: 2024-09-14T00:00:00.000Z
+        created: 2023-09-14T13:44:49.925Z
+  SNYK-JAVA-COMFASTERXMLJACKSONCORE-2421244:
```

Comment:
question: this looks like a standard dependency vulnerability [here](https://github.com/snyk/nexus-snyk-security-plugin/blob/0d39fef99da979e5c4afb98a7baffdf58d1bbfa4/pom.xml#L69-L71) that can be fixed by doing a patched version upgrade, is this not the case?

Reply:
it looks like a lot of the dependencies in the .snyk are down to the same dependency, is it not possible to bump the version [here](https://github.com/snyk/nexus-snyk-security-plugin/blob/0d39fef99da979e5c4afb98a7baffdf58d1bbfa4/pom.xml#L35) and resolve them?